Sources/Camille/main.swift

```diff
@@ -20,6 +20,16 @@ let bot = try SlackBot(
                     .makeChatPostMessage(target: im)
             }
         )),
+        TopicService(config: TopicServiceConfig(
+            userAllowed: { user in
+                return user.is_admin
+            },
+            warning: { channel, user in
+                return try SlackMessage()
+                    .line(user, " only admins can change the topic.")
```

Comment:
.line("I can't let you do that, ", user, ". Only admins are allowed to change the topic.") 🤖